processout-sdk/src/main/java/com/processout/processout_sdk/Network.java

```diff
@@ -127,7 +127,7 @@ public String getErrorType() {
 
     @Override
     public String toString() {
-        return "ErrorReponse{" +
+        return "ErrorResponse{" +
```

Comment:
What's this?

Reply:
It's just a generic function for this object (to print it)

Reply:
Is that the normal way of printing objects in Java?